lmms_eval/tasks/videomme/videomme.yaml

```diff
@@ -26,10 +26,13 @@ metric_list:
 lmms_eval_specific_kwargs:
   default:
     pre_prompt: ""
-    post_prompt: "\nAnswer with the option's letter from the given choices directly."
+    post_prompt: "The best answer is:"
```

Comment:
Maybe you can keep the default one unchange as you have added prompt for llava_vid individually